### PR TITLE
ERP trigger to receivers for Field Reports

### PR DESCRIPTION
**Pull request by GregoryHorvath**
Ref.: https://github.com/IFRCGo/go-frontend/issues/1799

## Changes
- Added receiver for Field Report create/update
- Added the logic for constructing and posting the data, with logger HTTP status error handling

### CODE REVIEW

**Review by thenav56 — COMMENTED** [review]
We should properly also add some mock tests where we compare payload snaphost which is sent to ERP_API_ENDPOINT.

File: requirements.txt [hunk 1]
```diff
@@ -39,6 +39,7 @@ django-extensions==2.0.6
 django-filter==2.2.0
 django-guardian==1.5.1
 django-modeltranslation==0.15
+#django-pdb==0.6.2
```
[the remark this answers is not in the document]
Comment (szabozoltan69) [reply requirements.txt:42]: We can eliminate it from here, but in that case I have to always search it in my notes, which version is it needed. Can I leave it this way?

File: utils/erp.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+
+# FIXME: this
+ERP_API_ENDPOINT = settings.ERP_API_ENDPOINT
+
+
+def log_errors(errors):
+    if len(errors):
+        logger.error('Produced the following errors:')
+        logger.error('[%s]' % ', '.join(map(str, errors)))
```
[the remark this answers is not in the document]
Comment (szabozoltan69) [reply utils/erp.py:17]: Yet.
When tests begin and there will be some erroneous workflows, we shall.

[the remark this answers is not in the document]
Comment (thenav56) [reply utils/erp.py]: Okay. We should only add some snapshot test for the payload send to ERP_API_ENDPOINT.

File: requirements.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (thenav56) [reply requirements.txt]: Okay

**Comment by szabozoltan69** [comment]
Deployed to staging

**Pull request merged**